content/it/docs/concepts/containers/images.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Immagini
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+L'immagine di un container rappresenta dati binari che incapsulano un'applicazione e 
+tutte le sue dipendenze software. Le immagini sono costituite da pacchetti software 
+eseguibili che possono essere avviati in modalità standalone e su cui si possono fare 
+ipotesi ben precise circa l'ambiente in cui vengono eseguiti.
+
+Tipicamente viene creata un'immagine di un'applicazione ed effettuato il _push_ 
+su un registro prima di poterne fare riferimento esplicito in un 
+{{< glossary_tooltip text="Pod" term_id="pod" >}}
+
+Questa pagina va a delineare nello specifico il concetto di immagine di un container.
+
+<!-- body -->
+
+## I nomi delle immagini
+
+Alle immagini dei container vengono normalmente attribuiti nomi come `pausa`, `esempio/ilmiocontainer`, o `kube-apiserver`.
+Le immagini possono anche contenere l'hostname del registro; ad esempio: `registro.fittizio.esempio/nomeimmagine`,
+ed è possibile che sia incluso nel nome anche il numero della porta; ad esempio: `registro.fittizio.esempio:10443/nomeimmagine`.
+
+Se non si specifica l'hostname di un registro, Kubernetes assume che ci si riferisca al registro pubblico di Docker.
+
+Dopo la parte relativa al nome dell'immagine si può aggiungere un _tag_ (come comunemente avviene per comandi come `docker` e `podman`).
+I tag permettono l'identificazione di differenti versioni della stessa serie di immagini.
+
+I tag delle immagini sono composti da lettere minuscole e maiuscole, numeri, underscore (`_`),
+punti (`.`), e trattini (`-`).  
+Esistono regole aggiuntive relative a dove i caratteri separatori (`_`, `-`, and `.`) 
+possano essere inseriti nel tag di un'immagine. 
+Se non si specifica un tag, Kubernetes assume il tag `latest` che va a definire l'immagine disponibile più recente.
+
+{{< caution >}}
+Bisognerebbe evitare di utilizzare il tag `latest` quando si rilasciano dei container in produzione,
+in quanto risulterebbe difficile tracciare quale versione dell'immagine sia stata avviata e persino più difficile
+effettuare un rollback ad una versione precente.
+
+Invece, meglio specificare un tag specifico come ad esempio `v1.42.0`.
+{{< /caution >}}
+
+## Aggiornamento delle immagini
+
+Quando un {{< glossary_tooltip text="Deployment" term_id="deployment" >}},
+{{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}, Pod, o qualsiasi altro
+oggetto che includa un Pod template viene creato per la prima volta, la policy di default per il pull di tutti i container nel Pod 
+è impostata su `IfNotPresent` (se non presente) se non specificato diversamente.
+Questa policy permette al
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} di evitare di fare il pull
+di un'immagine se questa è già presente.
+
+Si può forzare il pull:
```

Comment:
Se necessario, si può forzare il pull in ogni occasione in uno dei seguenti modi: